Adding terms to glossary (#28884)
grafana/grafana: docs/sources/getting-started/glossary.md

      Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
    </td>
  </tr>
  <tr>

>     <td style="vertical-align: top">Exporter</td>

Reviewer: If you add exporter, consider adding "export" as well. I'm concerned our readers will confuse this with the import/export dashboards functionality.

Author: Will do

    <td>
      An exporter implements the translation of data that comes out of a data source into a format that Prometheus can digest.
    </td>
  </tr>
  <tr>
    <td style="vertical-align: top">Integration (Grafana Cloud)</td>
    <td>

>       Each Integration in Grafana Cloud uses the cloud agent to connect your data source to Grafana Cloud for visualizing. Note: Prometheus uses the word “integrations” to refer to software that exposes Prometheus metrics without needing an exporter, which is a different use of the same word we use here.

Reviewer: I suggest asking @jhunthrop for his input, because there are several definitions of "integrations" in the Grafana context.

Author: I think this entry is pretty clear that it is in the Grafana Cloud context, with some disambiguation provided for the Prometheus context. If/when we learn more, splitting this entry into two and adding a third for how Grafana uses the term differently outside of cloud might be our best option.

Reviewer: Fair enough.

>       Each Integration in Grafana Cloud uses the cloud agent to connect your data source to Grafana Cloud for visualizing. Note: Prometheus uses the word “integrations” to refer to software that exposes Prometheus metrics without needing an exporter, which is a different use of the same word we use here.

Reviewer: Should Cloud Agent be capitalized?

Author: No. I'm trying to avoid the unnecessary proliferation of proper nouns when descriptive ones serve the purpose. I may lose that battle, but haven't today.

    </td>
  </tr>
  <tr>
    <td style="vertical-align: top">Graph</td>
    <td>
      A commonly-used visualization that displays data as points, lines, or bars.
    </td>
  </tr>
  <tr>

>     <td style="vertical-align: top">Mixin</td>

Reviewer: Please add "mixin" to the word usage section of the style guide.

    <td>

>       A mixin is a set of Grafana dashboards and Prometheus rules and alerts, written in jsonnet and packaged together in a bundle.

Reviewer: ```suggestion
      A mixin is a set of Grafana dashboards and Prometheus rules and alerts, written in Jsonnet and packaged together in a bundle.
```

      Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
    </td>
  </tr>
  <tr>

>     <td style="vertical-align: top">Export/Import</td>

Reviewer 2: ```suggestion
    <td style="vertical-align: top">Export/Import dashboard</td>
```

While I can't think of another resource that we "export"  or "import", it's a common terminology and is likely used or will be used for other things as well. Would it make sense to make it a little more specific to exporting/importing _dashboards_.

    <td>
      Grafana includes the ability to export your dashboards to a file containing JSON. Community members sometimes share their created dashboards on the [Grafana Dashboards page](https://grafana.com/grafana/dashboards). Dashboards previously exported or found on this site may be imported by other users.
    </td>
  </tr>
  <tr>
    <td style="vertical-align: top">Exporter</td>
    <td>

>       An exporter implements the translation of data that comes out of a data source into a format that Prometheus can digest.

Reviewer 2: ```suggestion
      An exporter translates data that comes out of a data source into a format that Prometheus can digest.
```

    </td>
  </tr>
  <tr>
    <td style="vertical-align: top">Integration (Grafana Cloud)</td>
    <td>

>       Each Integration in Grafana Cloud uses the cloud agent to connect your data source to Grafana Cloud for visualizing. Note: Prometheus uses the word “integrations” to refer to software that exposes Prometheus metrics without needing an exporter, which is a different use of the same word we use here.

Reviewer 2: Why is Integration capitalized? Is this a product name?

Author: Yeah, that's how it is being used at the moment. Not my ideal, but it does differentiate Grafana Cloud Integrations from other uses of the same word.
